### PR TITLE
Marketplace search fixes

### PR DESCRIPTION
**Pull request by skurfuerst**

### CODE REVIEW

**Comment by bwaidelich** [comment]
I can't really judge, but I don't see any obvious bugs and if it fixes the search for you I'm +100 ;)

**Review by daniellienert — APPROVED** [review]

**Comment by daniellienert** [comment]
Looks great. The problem caused by the getRequest method called twice should be also fixed.

File: Classes/Eel/ElasticSearchQueryBuilder.php [hunk 1]
```diff
@@ -76,10 +80,11 @@ public function fulltext($searchWord)
 
     /**
      * return void
+     * @param QueryInterface $request
      */
-    protected function skipAbandonnedPackages()
+    protected static function skipAbandonnedPackages(QueryInterface $request)
```
Comment (kdambekalns) [inline Classes/Eel/ElasticSearchQueryBuilder.php:85]: You could have fixed the name typo, but… 🤷‍♂️

**Comment by kdambekalns** [comment]
So it was me who broke it, probably (I adjusted it to the `QueryInterface` some days ago). Anyway, looks good, even after being merged already :)

**Pull request merged**